draft-ietf-tls-dtls-connection-id.md

```diff
@@ -83,26 +86,23 @@ application data protection. Ideally, the second phase where application data is
 protected lasts over a longer period of time since the established keys will only
 need to be updated once the key lifetime expires.
 
-In the current version of DTLS, the IP address and port of the peer is used to
+In the current version of DTLS, the IP address and port of the peer are used to
 identify the DTLS association. Unfortunately, in some cases, such as NAT rebinding,
 these values are insufficient. This is a particular issue in the Internet of Things
-when the device needs to enter extended sleep periods to increase the battery lifetime
-and is therefore subject to rebinding. This leads to connection failure, with the
-resulting cost of a new handshake.
+when devices enter extended sleep periods to increase their battery lifetime. The
+NAT rebinding leads to connection failure, with the resulting cost of a new handshake.
```

Comment:
In future it would be good to have different PRs for off-topic changes.

Reply:
ACK